go.mod

```diff
@@ -5,7 +5,7 @@ go 1.18
 require (
 	github.com/filecoin-project/go-address v1.1.0
 	github.com/filecoin-project/go-fil-commcid v0.1.0
-	github.com/filecoin-project/go-state-types v0.13.1
+	github.com/filecoin-project/go-state-types v0.14.0-dev.0.20240611104512-19a29aa50e4a
```

Comment:
I'd cut a release if we're ready.

Reply:
Agreed, it was just waiting for approval but I've given it a tick so @rjan90 can move forward there.

Reply:
Updated to go-state-types v0.14.0-rc1 in: https://github.com/filecoin-project/filecoin-ffi/pull/456/commits/ca6756413c0c8fded7b56d07b4fcc9a7c890afbb